data/sql_updates/create_omnibus_dates.sql

```diff
@@ -1,14 +1,36 @@
+drop materialized view if exists ofec_omnibus_dates_mv_tmp;
+
+-- pre-processing to get the right district-state combos for House members and
+-- the election doesn't exist if the status ID is not 1
+drop view if exists tmp_election_vw;
```

Comment:
Since we're already using common table expressions (`with subquery as (select ...)`), how about continuing that pattern instead of adding a temporary view?